BUILD

```diff
@@ -1,11 +1,75 @@
+load("@rules_python//python:defs.bzl", "py_runtime", "py_binary")
+load("@rules_cc//cc:defs.bzl", "cc_library")
+
+load("//:tools/psim.bzl", "psim_autocode")
+
+# Build flight software's gnc code as a library.
+cc_library(
+    name = "gnc",
+    srcs = glob([
+        "src/gnc/**/*.hpp", "src/gnc/*.cpp", "src/gnc/**/*.inl",
+        "include/gnc/**/*.inl"
+    ]),
+    hdrs = glob([
+        "include/gnc/**/*.hpp", "include/orb/*.h", "include/orb/*.hpp"
+    ]),
+    includes = ['include'],
+    copts = ['-Isrc'],
+    linkstatic = True,
+    visibility = ["//visibility:public"],
+    deps = ["@lin//:lin"],
+)
+
+# Let bazel know about our virtual environment.
+#
+# Also see additional options passed with .bazelrc to ensure we're using the
+# virtual environment.
+#
+# TODO : This is a little hacky and could be cleaned up with toolchains.
```

Comment:
Tbh not worth it.

Reply:
Agreed haha But just in case something breaks down the line I wanted to leave this note here.